src/excmds.ts

```diff
@@ -370,6 +370,58 @@ export function loggingsetlevel(logModule: string, level: string) {
 
 // {{{ PAGE CONTEXT
 
+//#content_helper
+export let JUMPLIST = []
+//#content_helper
+export let JUMPCURRENT: number
+//#content_helper
+export let JUMPTIMEOUT: number
+//#content_helper
+export let JUMPED: boolean
+
+//#content
+export function jumpnext(n = 1) {
+    jumpprev(-n)
+}
+
+/** Similar to Pentadactyl or vim's jump list.
+    Should be bound to <C-o> when modifiers are implemented
+*/
+//#content
+export function jumpprev(n = 1) {
+    JUMPCURRENT -= n
+    if (JUMPCURRENT < 0) return back(-JUMPCURRENT)
+    else if (JUMPCURRENT >= JUMPLIST.length) return forward(JUMPCURRENT - JUMPLIST.length + 1)
+    let p = JUMPLIST[JUMPCURRENT]
```

Comment:
If you want to quickly move through history with this bound to `<C-{i,o}>`, we get all the way to here as `JUMPLIST` has zero length, and then `p` often ends up undefined as the page hasn't been open for 1000ms yet and we get an error popping up in the command line.